### PR TITLE
Update screenshots in UG

### PR DESCRIPTION
**Pull request by lowkangn**
The screenshots look almost identical to the ones in the DG, just that the UI is updated.

### CODE REVIEW

File: docs/tutorials/TracingCode.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ When trying to understand an unfamiliar code base, one common strategy used is t
 
 Before we jump into the code, it is useful to get an idea of the overall structure and the high-level behavior of the application. This is provided in the 'Architecture' section of the developer guide. In particular, the architecture diagram (reproduced below), tells us that the App consists of several components.
 
-![ArchitectureDiagram](../images/ArchitectureDiagram.png)
+![ArchitectureDiagram](../images/DG%20images/ArchitectureDiagram.png)
```
Comment (XXJJXJ) [inline docs/tutorials/TracingCode.md:19]: What happened to this? 😅

**Review by XXJJXJ — APPROVED** [review]
Only 1 portion which I don't understand,else its good 👍

File: docs/tutorials/TracingCode.md (hunk 1 above)
Comment (lowkangn) [reply docs/tutorials/TracingCode.md:19]: what the

**Review by kanjitp — APPROVED** [review]
Got stuttered at the number of files at first

**Pull request merged**